app/services/container_deployment_service.rb

```diff
@@ -54,6 +54,6 @@ def optional_vms(vms)
   end
 
   def optional_deployment_types
-    %w(openshiftOrigin, openshiftEnterprise)
+    %w(openshiftOrigin openshiftEnterprise)
```

Comment:
@simon3z OK, how about we go with the way the types appear in the .cfg yaml:
["origin,", "openshift-enterprise"]

if that's OK I'll add it as a constant to ContainerDeployment model and then just:
```
def optional_deployment_types
  ContainerDeployment::DEPLOYMENT_TYPES
end
```

@alongoldboim